Coffee_loop/src/CoffeeLoop.java

```diff
@@ -0,0 +1,185 @@
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+public class CoffeeLoop {
+    public static void main(String[] args) throws IOException {
+        int countOfWater = 400;
+        int countOfMilk = 540;
+        int countOfCoffeeBeans = 120;
+        int countOfCups = 9;
+        int countOfMoney = 550;
+
+        int[] array = new int[5];
+        array[0] = countOfWater;
+        array[1] = countOfMilk;
+        array[2] = countOfCoffeeBeans;
+        array[3] = countOfCups;
+        array[4] = countOfMoney;
+
+        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
+
+        while(true) {
+            System.out.println("Write action (buy, fill, take, remain, exit):");
+            System.out.print(">");
+            String action = reader.readLine();
+
+            if (action.equals("buy")) {
```

Comment:
@AraChilingaryan method is too big , also you can use switch case here

Reply:
done